package.json

```diff
@@ -9,6 +9,12 @@
     "type": "git",
     "url": "git://github.com/Pana/nrm.git"
   },
+  "files": [
+    "actions.js",
+    "constants.js",
+    "helpers.js",
+    "registries.json"
```

Comment:
Add  "registries.json",  "cli.js", "process.js" to files.

Reply:
I had added `registries.json`. 

The  `cli.js` file will be added default. 

![image](https://github.com/user-attachments/assets/fcd8b7ae-2038-479e-8308-032b57b5d152)

The `process.js` may be a test file?

![image](https://github.com/user-attachments/assets/11ec79f8-81f2-4efb-965c-29223db28d7f)

Reply:
`process.js` is used by helpers.js

Reply:
ok, I see it. The comment mistakens me. I will update code.